src/layer/marker/Marker.js

```diff
@@ -205,17 +209,14 @@ export var Marker = Layer.extend({
 			}
 
 			if (icon.tagName === 'IMG') {
-				icon.alt = options.alt || '';
+				icon.alt = options.alt || 'Location';

```

Comment:
Instead of hardcoding some arbitrary string you better change `options.alt` defaults.

Reply:
"location" seems opinionated, and may not make sense in a non-geographic map. How about "Marker"?